WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -748,6 +758,65 @@ private boolean checkCredentials() {
         return true;
     }
 
+    public static class StatsWidgetPromoDialogFragment extends DialogFragment {
+
+        public static StatsWidgetPromoDialogFragment newInstance() {
+            StatsWidgetPromoDialogFragment frag = new StatsWidgetPromoDialogFragment();
```

Comment:
Minor nit: the `frag` variable serves no purpose here - this could simply be `return new StatsWidgetPromoDialogFragment();`